domains/arifsani.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "arifsans",
+        "email": "arifsanimail@gmail.com"
+    },
+    "record": {
+        "CNAME": ["arifsani.xyz"]
```

CNAME shouldn't be in an array
```suggestion
        "CNAME": "arifsani.xyz"
```